presto-hive/src/main/java/io/prestosql/plugin/hive/HiveModule.java

```diff
@@ -119,7 +119,7 @@ public void configure(Binder binder)
         fileWriterFactoryBinder.addBinding().to(RcFileFileWriterFactory.class).in(Scopes.SINGLETON);
 
         configBinder(binder).bindConfig(ParquetReaderConfig.class);
-        configBinder(binder).bindConfig(ParquetWriterConfig.class);
+        configBinder(binder).bindConfig(ParquetWriterConfig.class, "hive");
```

Comment:
Should I make the same change (drop `@LegacyConfig` and use `bindConfig` with `"hive"` in `HiveModule` instead of) to `ParquetReaderConfig`? Right now, the usage of the reader and writer properteis is inconsistent, as seen here.

But `ParquetReaderConfig` has been using `@LegacyConfig` for nearly a year, since 5ab85b7.

Reply:
i guess this is a question is to @electrum (per https://github.com/prestosql/presto/pull/5376#pullrequestreview-500411843)
or to @martint (per https://github.com/prestosql/presto/commit/5ab85b7263f8988589721735ea2a7ec625a1a949)